docs/specification/1.0.md

```diff
@@ -523,6 +524,36 @@ The following example illustrates a delete action:
 }
 ```
 
+
+
+#### OverrideReason
+
+An **OverrideReason** is described by the following attributes. This specification does not prescribe a standard set of override reasons; implementers are encouraged to submit suggestions for standardization. To enable analysis of a card's usage, the CDS service SHALL send the same `code`, `system`, and `label` across CDS Hooks requests for the same CDS client.
```

Comment:
"...across CDS Hooks **responses** for the same..."

> the CDS service SHALL send the same `code`, `system`, and `label` across CDS Hooks requests for the same CDS client.

I want to make sure this doesn't get read as each card must have the same code(s), but that the codes/systems/labels are from a _set_ of codes/systems/labels that is consistently drawn from across replies.  What about:

The CDS Service SHALL use a consistent set of override reasons across CDS Hooks responses for the same CDS Client

Reply:
I agree with the confusion; your proposed language here isn't helping me, though. I think we're trying to say: 

1. for a given CDS Client and label the CDS Service SHALL send the same `code` and `system`
2. for a given CDS Client, `code`, and `system`, the CDS Service SHALL send the same `label`

Is that right? i.e., both (1) and (2) are desired? Why is this important? What is the connection to analytics?